### PR TITLE
Bookingsync application1

### PR DESCRIPTION
**Pull request by Mandaryn**
dropping rails 4 as its hard to keep the dependencies resolve for both 4 and 5

### CODE REVIEW

**Comment by Fredar** [comment]
@Mandaryn Think we can remove gemfiles for 4.x then

File: app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def new
       def create
         BookingsyncPortal::Write::EnsureSourceExists.new(current_account).call
         @remote_account = scope.create(params_remote_account)
-        respond_with @remote_account, location: admin_rentals_url
+        respond_with @remote_account, location: admin_rentals_path
```
Comment (Fredar) [inline app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb]: whats the reason here, just curious

Comment (Mandaryn) [inline app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb]: in rails 5 the host is not automatically inferred from env, so those don't work with extra setup.
but it is not necessary to have full urls here, paths should be good enough?

Comment (Fredar) [inline app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb]: I guess paths are ok, just wondered about the trigger for this change

**Comment by Fredar** [comment]
For some reason its still running the old gemfiles and not includin the 5.1 one.
Maybe add ruby 2.4 as well?

**Comment by Fredar** [comment]
@Mandaryn we need migrations with tags to work with 5.1 rails

**Pull request merged**